### DISCUSSION TITLE
Hue grammars online editor/tester

### DISCUSSION THREAD

**Discussion by mingodad** [cloudera/hue#3473] [Ideas]
I've just added `hue-sql-generic` grammar to https://mingodad.github.io/parsertl-playground/playground/ an `Yacc/Lex` compatible online editor/tester (select `Hue SQL generic parser` from `Examples` then click `Parse` to see a parser tree for the content in `Input source` editor).

I hope that it can help debug/develop/document/test this project grammars !

Any feedback is welcome !

**Comment by bjornalm** [comment]
FYI @JohanAhlen
